provider/cloudflare/cloudflare.go

```diff
@@ -495,18 +470,18 @@ func (p *CloudFlareProvider) submitCustomHostnameChanges(ctx context.Context, zo
 }
 
 // submitDataLocalizationRegionalHostnameChanges applies a set of data localization regional hostname changes, returns false if it fails
-func (p *CloudFlareProvider) submitDataLocalizationRegionalHostnameChanges(ctx context.Context, changes []DataLocalizationRegionalHostnameChange, resourceContainer *cloudflare.ResourceContainer) bool {
+func (p *CloudFlareProvider) submitDataLocalizationRegionalHostnameChanges(ctx context.Context, changes []regionalHostnameChange, resourceContainer *cloudflare.ResourceContainer) bool {
```

Comment:
am i missing any specific consideration of not moving it to the `cloudflare_regional.go` too?

nitpick: `change/changes` variables is of `regionalHostnameChange` type in this function and `cloudFlareChange` everywhere else, including following`dataLocalizationRegionalHostnamesChanges()`. As long as it's syntactically correct, we might prefer to name it distinctly different to avoid confusion when reading code, wdyt?

Reply:
cf. https://github.com/kubernetes-sigs/external-dns/pull/5329#discussion_r2067042033.

For your second point you means renaming `change/changes` to `rhChange/rhChanges` for example ?

Reply:
yes, anything like it

Reply:
I move it to `cloudflare_regional.go` and renamed `change` to `rhChange`